dev/ci/ci-vst.sh

```diff
@@ -11,9 +11,9 @@ if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
 
 export COMPCERT=bundled
 
-ulimit -s
-ulimit -s 65536
-ulimit -s
+sudo ulimit -s
+sudo ulimit -s 65536
+sudo ulimit -s
 ( cd "${CI_BUILD_DIR}/vst"
   make IGNORECOQVERSION=true
 )
```

Comment:
Same comment as in #17903.

Reply:
That was debugging code introduced to locally run `make ci-vst`. It shoud not have been committed. Thanks for noticing. Fixed.